index.html

```diff
@@ -495,14 +495,17 @@ <h3>A Special Time</h3>
 
               <form action="./" class="margin-y-md">
                 <div class="flex-container">
+                  <label>Name</label>
                   <input class="width100 margin-y-md" type="text" placeholder="Name" required>
                 </div>
 
                 <div class="flex-container">
+                  <label>E-mail</label>
                   <input class="width100 margin-y-md" type="email" placeholder="Email address" required>
                 </div>
 
                 <div class="flex-container">
+                  <label>Comment</label>
                   <textarea class="width100 margin-y-md" placeholder="Type your comment here" id="comment"
```

Comment:
You should add form validation for the comment field a comment can have up to 250 chars